README.md

```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
 yarn
 ```
 
+### Environment variables
+
+To get started local development, create a file in the root of the repo named
+`.env` with the following content.
+
+```bash
+GITHUB_PERSONAL_ACCESS_TOKEN = <insert personal access token here>
```

Comment:
```bash
GITHUB_PERSONAL_API_TOKEN=<your_github_api_token>
```

Reply:
Ok, i will change the env name in other PR then.